perfkitbenchmarker/configs/option_decoders.py

```diff
@@ -187,33 +187,38 @@ class FloatDecoder(TypeVerifier):
     min: None or float. If provided, it specifies the minimum accepted value.
   """
 
-  def __init__(self, component, option, max=None, min=None, **kwargs):
-    super(FloatDecoder, self).__init__(component, option, (float,), **kwargs)
+  def __init__(self, option, max=None, min=None, **kwargs):
+    super(FloatDecoder, self).__init__(option, (float,), **kwargs)
```

Comment:
Sorry, I wasn't aware when this class was introduced, so it was my changes that broke it.
I have a concern about the use of (float,) here. Because of the way yaml.load works, an input like 'docker_cpus: 2' will become an int that I don't think would be accepted by this FloatDecoder. It's probably best to accept int as well. Consider adding a test case to tests/configs/option_decoders_test.py